specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/textanalytics.json

```diff
@@ -0,0 +1,2254 @@
+                },
+                "validDocumentsCount": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "Number of valid documents. This excludes empty, over-size limit or non-supported languages documents."
+                },
+                "erroneousDocumentsCount": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "Number of invalid documents. This includes empty, over-size limit or non-supported languages documents."
+                },
+                "transactionsCount": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "Number of transactions for the request."
+                }
+            },
+            "description": "if showStats=true was specified in the request this field will contain information about the request payload."
+        },
+        "MultiLanguageBatchInput": {
+            "type": "object",
+            "description": "Contains a set of input documents to be analyzed by the service.",
+            "required": [
+                "documents"
+            ],
+            "properties": {
+                "documents": {
+                    "type": "array",
+                    "description": "The set of documents to process as part of this batch.",
+                    "items": {
+                        "$ref": "#/definitions/MultiLanguageInput"
+                    }
+                }
+            }
+        },
+        "MultiLanguageInput": {
+            "type": "object",
+            "description": "Contains an input record to be analyzed by the service.",
+            "required": [
+                "id",
+                "text"
+            ],
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "A unique, non-empty record identifier."
+                },
+                "text": {
+                    "type": "string",
+                    "description": "The input text to process."
+                },
+                "language": {
+                    "type": "string",
+                    "description": "(Optional) This is the 2 letter ISO 639-1 representation of a language. For example, use \"en\" for English; \"es\" for Spanish etc. If not set, use \"en\" for English as default."
+                }
+            }
+        },
+        "ErrorResponse": {
+            "type": "object",
+            "description": "Error response.",
+            "additionalProperties": false,
+            "properties": {
+                "error": {
+                "description": "The error object.",
+                    "$ref": "#/definitions/Error"
+                }
+            }
+        },
+        "Error": {
+            "type": "object",
+            "description": "The error object.",
+            "additionalProperties": false,
```

Comment:
Azure allows additional properties in the Error object as this is a good place for services to provide additional structured information about an error.  However, if you have already GA'ed with `additionalProperties: false` then you probably can't change it.

Reply:
This API won't GA until April, so I removed `additionalProperties` from `Error` and `ErrorResponse`.